damnit/backend/db.py

```diff
@@ -218,6 +225,12 @@ def update_computed_variables(self, vars: dict):
                 for (n, v) in updates.items()
             ])
 
+            if not set(vars) <= set(vars_in_db):
```

Comment:
I don't think that's quite what we want, because we can have a computed variable - with data - that was previously in the context file but no longer is, e.g. because the function was renamed. I assume we wouldn't automatically delete the old data from the table in that case.

Reply:
Oops, yes you're quite right :grimacing: